hosts

```diff
@@ -1,2 +1,4 @@
-firstone ansible_ssh_host=192.168.121.182 ansible_ssh_user=vagrant
-secondone ansible_ssh_host=192.168.121.199 ansible_ssh_user=vagrant
+[alpha]
+firstone ansible_ssh_host=192.168.121.33 ansible_ssh_user=vagrant
+secondone ansible_ssh_host=192.168.121.103 ansible_ssh_user=vagrant
+thirdone ansible_ssh_host=192.168.124.55 ansible_ssh_user=kbattula
```

Comment:
How is this file used? Generally, Does it generally need the kbattula user in it?